### ISSUE TITLE
1.0.0 Release checklist

### ISSUE THREAD

**Issue by iMicknl** [BotBuilderCommunity/botbuilder-community-js#90]
## Post 1.0.0 release
- [x] Close #83
- [x] Deprecate [@botbuildercommunity/adapters](https://www.npmjs.com/package/@botbuildercommunity/adapters) and refer to `@botbuildercommunity/adapter-console`.
- [x] Review all `CHANGELOG.md` files and add publication date.

**Comment by szul** [comment]
@iMicknl I was going to wait on deprecating the old adapters package and pointing to the new one until after v1 is merged into master. Let me know if you want to handle this differently.

**Comment by iMicknl** [comment]
@szul deprecating the old packages is stated for post 1.0.0 release.

**Comment by szul** [comment]
Once #99 is merged, all packages should have changelogs, and every package except the ATS storage package will conform to the ESLint rules. What more do you think is needed before merging v1 with master and performing the post v1 items?

**Comment by szul** [comment]
@iMicknl I think we're in a good state to merge v1 with master. The console adapter and ATS storage have been broken out/renamed, and we have the renamed middleware packages. Everything except for the ATS storage is passing ESLint, and the Azure Pipeline build checks are solid. We have changelogs in each as well. I'd like to get this merged so we can focus on the requested adapters and some other items without worrying about merge backs, future deprecations (because of renames), etc.

**Comment by iMicknl** [comment]
@szul, fine by me! Maybe we could have a look for automating the releases in the future, after merge to master and all tests pass.

I will be on vacation until October 4, so don’t wait for my comments. 😉
